modules/10_pull_request_pipeline/README.md

```diff
@@ -0,0 +1,95 @@
+# Pull request pipeline
+
+In this stage we'll be using [Github Actions](https://docs.github.com/en/actions/learn-github-actions/understanding-github-actions) to create a pull request pipeline that will run your tests and lint your project on each [pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests) made to your repository. A pipeline in Github Actions is called a [workflow](https://docs.github.com/en/actions/using-workflows/about-workflows).
```

Comment:
```suggestion
In this stage we'll add automation to the project. We're going to automatically run tests and lint the code every time one makes a [pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests)(PR) to your repository. This will help us to catch bugs early and keep the code clean.

People usually call such an automation a "pipeline" or a "workflow", and running quality checks on a PR is an important part of so called "Continuous Integration", or simply "CI". 

If your code is hosted on GitHub, a simple way to create a workflow is to use [Github Actions](https://docs.github.com/en/actions/learn-github-actions/understanding-github-actions). This is what we will do in this tutorial.
```